modules/internal/Bytes.chpl

```diff
@@ -306,6 +306,34 @@ module Bytes {
       }
     }
 
+    proc chpl__serialize() {
```

Comment:
Should these have "no doc"? I didn't see them in `String.chpl` nor do I see them in the generated docs but when I attempted this for regex Elliot suggested adding a no doc.

Reply:
They don't need to. This is a primary method declared inside a record that has `no doc`. Nothing within the record definition will have documentation generated.